manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 3,
   "name": "Hide LinkedIn Applicants",
-  "version": "0.0.9",
+  "version": "0.0.10",
```

Comment:
I'm using semantic versioning but debating if this should be 0.0.10 or 0.1.0. The Chrome Web Store also has specific requirements so I'd like these numbers to be aligned.